Automattic/wp-calypso
# Repeat Visitor: add additional block keywords

#### Changes proposed in this Pull Request

Based on feedback in our internal call for testing (p1HpG7-6k7-p2), this PR adds additional keywords to help new users discover the Repeat Visitor block.

#### Testing instructions

Spin up a [Jurassic Ninja instance](https://jurassic.ninja/create/?gutenpack&calypsobranch=update/repeat-visitor-keywords).

Connect Jetpack to your WPCOM account.

Navigate to Settings->Jetpack Constants, check JETPACK_BETA_BLOCKS, and Save.

Navigate to the post editor and click + to add a new block. Try searching for a block with the following terms and make sure that Repeat Visitor is suggested:
* visitor
* hidden
* conditional
